tests/api2/test_alert.py

```diff
@@ -0,0 +1,86 @@
+from time import sleep
+
+import pytest
+from pytest_dependency import depends
+
+from auto_config import pool_name
+from middlewared.test.integration.utils import call, ssh
+
+
+def get_alert_by_id(alert_id):
+    results = call("alert.list")
+    for alert in results:
+        if alert['id'] == alert_id:
+            return alert
+
+
+def alert_exists(alert_id):
+    return isinstance(get_alert_by_id(alert_id), dict)
+
+
+@pytest.mark.dependency(name='degrade_pool')
+def test_degrading_a_pool_to_create_an_alert(request):
+    get_pool = call("pool.query", [["name", "=", pool_name]])[0]
```

Comment:
You can pass `[["name", "=", pool_name]], {"get": True}` and `get_pool` will be returned so you don't have to do `[0]`